tests/common/fixtures/services.py

```diff
@@ -0,0 +1,25 @@
+from unittest.mock import create_autospec
+
+import pytest
+
+from h.services.search_index import SearchIndexService
+
+# Allow people to mass import everything without getting things we don't
+# mean them to like pytest.
+__all__ = ('service_mocker', 'search_index')
+
+
+@pytest.fixture
+def service_mocker(pyramid_config):
+    def service_mocker(class_, name):
+        service = create_autospec(class_, instance=True)
+        pyramid_config.register_service(service, name=name)
+
+        return service
+
+    return service_mocker
+
+
+@pytest.fixture
+def search_index(service_mocker):
+    return service_mocker(SearchIndexService, "search_index")
```

Comment:
I think it would be nice to collect all of these re-usable fixtures here

Reply:
Yeah, I think this is a great idea. Same as we have in LMS